main.cpp

```diff
@@ -52,84 +52,94 @@ arm.rotateCounterClockwise();           // 4) demonstrates CamelCase
 
 
 //  1)
-//  Noun:
-//  action 1:
-
-//  action 2:
+//  Noun: car
+//  action 1: car moves Forward
+car.moveForward();
+//  action 2: car ignition turns on
+car.ignitionOn();
+//  action 3: car headlights turn off
+ car.headlightsOff();
 
-//  action 3:
- 
 //  2)
-//  Noun:
-//  action 1:
-
-//  action 2:
+//  Noun: ship
+//  action 1: ship sounds honk
+ship.honk();
+//  action 2: ship raises flag
+ship.flagRaise();
+//  action 3: ship takes anchor away
+ ship.anchorAway();
 
-//  action 3:
- 
 //  3)
-//  Noun:
-//  action 1:
+//  Noun: window
+//  action 1: window breaks
+window.breaks();
+//  action 2: window goes dark
+window.goDark();
+//  action 3: window closes vertically
+ window.closeVertically();
 
-//  action 2:
-
-//  action 3:
- 
 //  4)
-//  Noun:
-//  action 1:
-
-//  action 2:
+//  Noun: chair
+//  action 1: chair holds on two feet
+chair.holdTwoFeet();
+//  action 2: chair makes weird noises
+chair.makeNoise();
+//  action 3: chair falls backwards
+ chair.backwardsFall();
 
-//  action 3:
- 
 //  5)
-//  Noun:
-//  action 1:
-
-//  action 2:
+//  Noun: hand
+//  action 1: hand turns red
+hand.turnsRed();
+//  action 2: hand dissapears
+hand.dissapears();
+//  action 3: hand makes fist
+ hand.makesFist();
 
-//  action 3:
- 
 //  6)
-//  Noun:
-//  action 1:
-
-//  action 2:
+//  Noun: keyboard
+//  action 1: keyboard LED lights on
+keyboard.lightOn();
+//  action 2: keyboard ESC key turns off
+keyboard.escOff();
+//  action 3: keyboard touch turns ON
+ keyboard.touchOn();
 
-//  action 3:
- 
 //  7)
-//  Noun:
-//  action 1:
+//  Noun: guitar (Electric)
+//  action 1: guitar string vibrate fast
+guitar.stringVibrateFast();
+//  action 2: guitar volume knob turns off
+guitar.volKnobOff();
+//  action 3: guitar headstock breaks down
+ guitar.headstockBreak();
```

Comment:
There are 2 problems here:
1) your plain english expressions don't use the same noun each time.
action 1: the noun in the expression is `guitar string`, not `guitar`.  the guitar string is what is vibrating.
action 2: the noun is `guitar volume knob`, not `guitar`.
action 3: the noun is `guitar headstock`, not `guitar`.

2) these don't match the plain-english expressions:
what you want to do is structure your english expressions as:
[noun] [verb] [verb target] [verb description]
Plain english:  `guitar vibrates the string fast`
pseudo-code: `guitar.vibrateStringFast();` 
Plain english: `guitar turns volume knob off`
pseudo-code: `guitar.turnVolumeKnobOff();`
plain english: `guitar breaks headstock down`
pseudo-code: `guitar.breaksHeadstockDown();`

This is a very subtle difference.  it is very important to learn this concept when you are starting to write your own code.  This skill will make your code very easy to understand.  
The concept I want you to understand is to name things what they are or what they do, which starts with being able to explain it using very simple plain english.
- [ ] revise